kuma/settings/common.py

```diff
@@ -1632,6 +1632,9 @@ def parse_iframe_url(url):
 ES_INDEXING_TIMEOUT = 30
 ES_LIVE_INDEX = config('ES_LIVE_INDEX', default=False, cast=bool)
 ES_URLS = config('ES_URLS', default='127.0.0.1:9200', cast=Csv())
+# The default 200 is because searching for something using more than 200
+# characters should not be necessary.
```

Comment:
Try not to mention the default in the code comment. If we decide to switch from 200 to 250, you'd need to edit in 3 places. 
Also, another truth about this is to avoid unnecesssary burden on Elasticsearch for queries that are probably mistakes or plainly just junk.